assets/chat/js/chat.js

```diff
@@ -294,7 +294,6 @@ class Chat {
             (a['alias'] || []).forEach(k => this.autocomplete.add(`/${k}`))
         });
         this.emoticons.forEach(e => this.autocomplete.add(e, true))
-        this.twitchemotes.forEach(e => this.autocomplete.add(e, true))
```

Comment:
Why this change?

Reply:
I kinda abused the fact that twitchemotes is empty right now so I used it for the suffix emotes. It should get renames to suffixemotes or something like that.

Reply:
So this removal is just cleanup? Sounds good in that case 👍